packages/components/src/Well/_docs/Well.mdx

```diff
@@ -24,11 +24,18 @@ Wells are used as a visual container around secondary content.
 
 ## API
 
-### Variants
-<Canvas of={WellStories.Variants} />
+### Color
+
+Formerly controlled by `variant`, `color` BorderStyles the background and border color of the `Well`.
```

Comment:
I take it `BorderStyles` shouldn't be there 😂 To revise sentence.

Reply:
Ha yeah, no idea what happened there, maybe a find a replace 😉